docs/send-data/opentelemetry-collector/remote-management/source-templates/syslog.md

```diff
@@ -0,0 +1,62 @@
+---
+id: syslog
+title: Syslog Source Template
+sidebar_label: Syslog
+description: Learn about the Sumo Logic Syslog source template for OpenTelemetry.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="30"/>
+
+The Syslog source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent, agent will start listening to the configured port for syslogs and send it to Sumo Logic.
+		
+## Fields creation in Sumo Logic for Syslog
+
+If not already present, the following [Fields](/docs/manage/fields/) are created as part of source template creation.
+
+- **`sumo.datasource`**. Fixed value of **localfile**.
+- **`deployment.environment`** User configured field at the time of collector installation. This identifies the environment where the host resides. For example: dev, prod or qa.
+- **`host.group`**. This is a collector level field and is user configured (at the time of collector installation). Through this, the group of host are identified.
+- **`host.name`**. This is tagged through the resourcedetection processor. It holds the value of the host name where the OTel collector is installed.
+
+## Prerequisitive
+Ensure that the syslog are in the [rfc5424](https://datatracker.ietf.org/doc/html/rfc5424) protocol. Since we use otel [syslog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/syslogreceiver) with this protocol. This will ensure proper parsing of the syslog metadata when ingested in Sumo Logic.
```

Comment:
```suggestion
Ensure that the syslogs conform to the [RFC 5424](https://datatracker.ietf.org/doc/html/rfc5424) protocol. Since we use the OpenTelemetry [syslog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/syslogreceiver) with this protocol, this will ensure proper parsing of the syslog metadata when ingested into Sumo Logic.
```